module.nix

```
@@ -109,20 +109,20 @@ in
           "uptrack-prefetch.service"
           "uptrack.service"
         ];
+        RefuseManualStop = "yes";
         StartLimitInterval = "90";
-        StartLimitBurst = "4";
       };
       serviceConfig = {
-        Type = "forking";
-        ExecStart = "${cfg.package}/bin/sentinelctl control run";
+        Type = "exec";
+        ExecStart = "${cfg.package}/bin/sentinelone-agent";
         WorkingDirectory = "/opt/sentinelone/bin";
         SyslogIdentifier = "${cfg.dataDir}/log";
-        WatchdogSec = "5s";
+        WatchdogSec = "30s";
         Restart = "always";
```

Given the latest state of the service file within the SentinelOne package, would it not make sense to update this to:
```suggestion
        Restart = "on-failure";
```